### PR TITLE
Download from server

### LINKED ISSUE #19: Cannot download .tgz
Use-case: To share the whole code to audience (fixed by #12) I've put a src.tar.gz in a directory and trainees' browsers would never allow download for security reason.

I separate this from #12 because we may want to share anything and this issue may appear with other files (I guess it depends on file extension).

### PR DESCRIPTION
**Pull request by naholyr**
Make the "download" a link to a real URL handled by copycast server.
- Pros: fixes #19 
- Cons: requires extra roundtrips to server

Discuss :)

### CODE REVIEW

**Comment by naholyr** [comment]
Ping @Delapouite for review

File: server/http.js [hunk 1]
```diff
@@ -33,10 +35,40 @@ const zipper = (tree, req, res, next) => {
...
+			res.setHeader('Content-Type', 'application/octet-stream')
+			res.setHeader('Content-Disposition', 'attachment; filename=' + path.basename(filename))
+			res.setHeader('Content-Transfer-Encoding', 'binary')
+			res.setHeader('Expires', '0')
+			res.setHeader('Cache-Control', 'must-revalidate, post-check=0, pre-check=0')
+			res.setHeader('Pragma', 'public')
+		}
+	})
+
+	const fullPrefix = '/' + prefix.replace(/^\/?(.*?)\/?$/, '$1') + '/'
```
Comment (Delapouite) [inline server/http.js:56]: The download from the server works great when I try to download `package.json` but fails for `.eslintrc.json`. I suspect there's something going on with hidden files. Maybe this regex?

Comment (naholyr) [inline server/http.js:56]: Looks like not, but I guess dotfiles and serve-static do not work well together

Comment (naholyr) [inline server/http.js:56]: Fixed by be309f6

**Pull request merged**